docs/Developer-guide/Building-GlusterFS.md

```diff
@@ -187,6 +188,14 @@ use `--enable-asan` option
 The above option will build with `-fsanitize=address -fno-omit-frame-pointer`
 options and uses the libasan.so shared library, so that needs to be available.
 
+
+In order to build GlusterFS without io-uring,
```

Comment:
`io_uring` is introduced on Linux kernel version 5.1. GlusterFS also needs the user space `liburing` helper library. If these are not available for your machine or if you wish to build GlusterFS without `io_uring` support,  use `--disable-linux-io_uring` option